extra/Lamdera/Evergreen/Snapshot.hs

```diff
@@ -648,6 +648,8 @@ canonicalToFt version scope interfaces recursionSet canonical tvarMap =
 
 
         -- Kernel concessions for Frontend Model and Msg
+        -- @TODO: Not sure if this should be here or not
+        ("elm", "browser", "Browser.Navigation", "Key") -> ("Browser.Navigation.Key", Set.singleton moduleName, Map.empty)
```

Comment:
I need help here, I know too little about snapshots. Note that I commented out the code for Browser.Navigation.Key in Wire3/Decoder.hs, Wire3/Encoder.hs and Wire3/Helpers.hs on purpose, while I kept it in TypeHash.hs. See the comments at the end of Injection.hs for why.